app/validators/changed_validator.rb

```diff
@@ -0,0 +1,7 @@
+class ChangedValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    return if record.action != 'update' && (value.blank? || value.saved_changes?)
```

Comment:
This code `value.blank?` is to verify is `nil` before check `.saved_changes` right? But if you use `||` the code will try `.saved_changes?` from a nil class, use `&&` or get back when you was using `value&.saved_changes?`

Reply:
Thanks!